common/travis/license.lst

```diff
@@ -171,6 +173,7 @@ EUPL-1.2
 Entessa
 ErlPL-1.1
 Eurosym
+FDK-AAC
```

Comment:
do you think we could switch fdk-aac package to this license string in this PR?

Reply:
Done.

@leahneukirchen